src/localization/modalStrings.json

```diff
@@ -65,7 +65,7 @@
     "location_service_comment": "Required to connect to bluetooth sensors",
     "bluetooth_permission": "Bluetooth",
     "bluetooth_permission_comment": "Required to connect to bluetooth sensors",
-    "permissions_intro_1": "The following permissions services are required by various features of the cold chain functionality.",
+    "permissions_intro_1": "The following permissions and services are required by various features of the cold chain functionality.",
```

Comment:
I liked the replacement suggestion 

`"The following permissions and services are required for cold chain functionality." `

as it was less verbose, but the missing and is probably the most important one for sure 🤣

Reply:
0915ad6ed8bf28abf3ac5adece7475c7b3a5daa6.